pkg/handler3/handler.go

```diff
@@ -40,8 +40,9 @@ import (
 )
 
 const (
-	subTypeProtobuf = "com.github.proto-openapi.spec.v3@v1.0+protobuf"
-	subTypeJSON     = "json"
+	subTypeProtobufDeprecated = "com.github.proto-openapi.spec.v3@v1.0+protobuf"
+	subTypeProtobuf           = "com.github.proto-openapi.spec.v3.v1.0+protobuf"
```

Comment:
cc @apelisse @liggitt on the naming of this header. It's the most consistent with what we already had, although things like discovery use `vnd.kubernetes.protobuf`.

Reply:
If I understand correctly, the new Content-Types will be:

`application/com.github.proto-openapi.spec.v2.v1.0+protobuf` and 
`application/com.github.proto-openapi.spec.v3.v1.0+protobuf`

```
Content-Type := type "/" subtype *[";" parameter] 

subtype := token

token := 1*<any CHAR except SPACE, CTLs, or tspecials> 

tspecials :=  "(" / ")" / "<" / ">" / "@"  ; Must be in 
           /  "," / ";" / ":" / "\" / <">  ; quoted-string, 
           /  "/" / "[" / "]" / "?" / "."  ; to use within 
           /  "="                        ; parameter values
```

~~According to the RFC Jordan posted in the other issue `.` is also a `tspecial` so it should not appear as part of `subtype`. Is this something we need to worry about? I see `vnd.kubernetes.protobuf` already breaks that rule.~~

Looking at this [MDN list of known/standard MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types), it seems the `vnd` reverse-DNS style is commonly used.

Since there is no official/standard way of expressing protobuf MIME types, if we can I would prefer consistency with the already well-known `vnd.kubernetes.protobuf`. It might make sense to include a param that includes which proto spec to use for example to delineate the v2/v3 protobuf types. Unless there is a reason to use different types for OpenAPI?

Reply:
>  I would prefer consistency with the already well-known application/vnd.kubernetes.protobuf

hmm... the `application/vnd.kubernetes.protobuf` mime-type implies a magic prefix of four bytes and wrapping in a `runtime.Unknown` message, which is how we serialize all our API objects... I was assuming we would want to keep the bytes we returned for this fixed mime-type identical to the problematic mime-type

Reply:
Ah did not realize, thats a good point. If the byte format is different that is a good reason to keep a separate mime-type.